deployment/data-streams/state.go

```diff
@@ -7,15 +7,17 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 
 	"github.com/smartcontractkit/chainlink/deployment"
-	"github.com/smartcontractkit/chainlink/deployment/common/types"
+	"github.com/smartcontractkit/chainlink/deployment/data-streams/changeset/types"
 	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/llo-feeds/generated/channel_config_store"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/llo-feeds/generated/verifier_proxy_v0_5_0"
```

Comment:
Should this import the non versioned `verifier_proxy` package?

Reply:
The interface is exactly the same for verifier proxy - but technically they're 2 different implementations / different source codes. The non-versioned go bindings are implicit to the v03/v04 contract versions.

For the verifier proxy this would work to use either - but for other contracts we need to specify the versioned go bindings